README.md

```diff
@@ -196,6 +195,8 @@ Example of `hook --config`:
 }
 ```
 
+> Note: watch for CRD objects is also possible, just use proper values for `apiVersion` and `kind` fields.
```

Comment:
>Note: watch for CRD objects is also possible, just use proper values for `apiVersion` and `kind` fields.

Note: watch for CRD objects is possible, just use proper values for `apiVersion` and `kind` fields.

Reply:
>CRD objects

Nope, that is semantically incorrect.

Reply:
Note: it is possible to watch custom resources, just use proper values for apiVersion and kind fields.

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^